src/apps/ociolutimage/main.cpp

```diff
@@ -305,12 +305,18 @@ int main (int argc, const char* argv[])
 // TODO: These should be exposed from inside OCIO, in appropriate time.
 //
 
-inline int GetLut3DIndex_RedFast(int indexR, int indexG, int indexB,
```

Comment:
The changes below this point are reverting some updates from PR #535. Can you update your branch from master and do another commit? Thanks.

Reply:
Sorry for the oversight